elastic/kibana
# [Maps] Allow implementers of ITooltipProperty to return React-DOM for the tooltip-value

Fixes https://github.com/elastic/kibana/issues/126631

To test
* Install sample data set
* Edit data view, adding "Color" or "URL" format for a field. This will render HTML
* Create a map, add a document layer, and add HTML formatted field to tooltip
* Open tooltip and ensure its still rendered as HTML

<img width="355" alt="Screen Shot 2022-03-07 at 9 58 39 AM" src="https://user-images.githubusercontent.com/373691/157107939-58a0b045-1191-4a45-9afe-e36d4657a1cd.png">
